src/GovUk.Education.ExploreEducationStatistics.Data.Model/GovUk.Education.ExploreEducationStatistics.Data.Model.csproj

```diff
@@ -11,6 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
+	<PackageReference Include="GeoJSON.Net" Version="1.2.19" />
```

Comment:
Looks like an alignment issue here or in this file as it's not formatted in-line with the other elements, but also see the comment I left in Admin.csproj about using `GeoJSON.Text` instead.

Reply:
Just checked and I think the issue is with a tab character that's crept in here.